From a083e4d4786357d06cfa2496926306d6a862d32e Mon Sep 17 00:00:00 2001
From: Bruno Bottazzini <bruno.bottazzini@intel.com>
Date: Wed, 6 Apr 2016 13:59:37 -0300
Subject: [PATCH] Do not install man docs

Signed-off-by: Bruno Bottazzini <bruno.bottazzini@intel.com>
---
 Makefile           |  1 -
 man/CMakeLists.txt |  5 -----
 man/Makefile       | 13 -------------
 3 files changed, 19 deletions(-)

diff --git a/Makefile b/Makefile
index 967ba26..e69d5d6 100644
--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,6 @@ test : mosquitto
 
 install : mosquitto
 	set -e; for d in ${DIRS}; do $(MAKE) -C $${d} install; done
-	set -e; for d in ${DOCDIRS}; do $(MAKE) -C $${d} install; done
 	$(INSTALL) -d ${DESTDIR}/etc/mosquitto
 	$(INSTALL) -m 644 mosquitto.conf ${DESTDIR}/etc/mosquitto/mosquitto.conf.example
 	$(INSTALL) -m 644 aclfile.example ${DESTDIR}/etc/mosquitto/aclfile.example
diff --git a/man/CMakeLists.txt b/man/CMakeLists.txt
index f0a9cf7..e69de29 100644
--- a/man/CMakeLists.txt
+++ b/man/CMakeLists.txt
@@ -1,5 +0,0 @@
-install(FILES mosquitto_passwd.1 mosquitto_pub.1 mosquitto_sub.1 DESTINATION ${MANDIR}/man1)
-install(FILES libmosquitto.3 DESTINATION ${MANDIR}/man3)
-install(FILES mosquitto.conf.5 DESTINATION ${MANDIR}/man5)
-install(FILES mosquitto-tls.7 mqtt.7 DESTINATION ${MANDIR}/man7)
-install(FILES mosquitto.8 DESTINATION ${MANDIR}/man8)
diff --git a/man/Makefile b/man/Makefile
index a049686..103b162 100644
--- a/man/Makefile
+++ b/man/Makefile
@@ -12,19 +12,6 @@ reallyclean : clean
 dist : mosquitto.8 mosquitto-tls.7 mosquitto.conf.5 mosquitto_passwd.1 mosquitto_pub.1 mosquitto_sub.1 mqtt.7 libmosquitto.3
 
 install :
-	$(INSTALL) -d ${DESTDIR}$(mandir)/man8
-	$(INSTALL) -m 644 mosquitto.8 ${DESTDIR}${mandir}/man8/mosquitto.8
-	$(INSTALL) -d ${DESTDIR}$(mandir)/man5
-	$(INSTALL) -m 644 mosquitto.conf.5 ${DESTDIR}${mandir}/man5/mosquitto.conf.5
-	$(INSTALL) -d ${DESTDIR}$(mandir)/man1
-	$(INSTALL) -m 644 mosquitto_passwd.1 ${DESTDIR}${mandir}/man1/mosquitto_passwd.1
-	$(INSTALL) -m 644 mosquitto_pub.1 ${DESTDIR}${mandir}/man1/mosquitto_pub.1
-	$(INSTALL) -m 644 mosquitto_sub.1 ${DESTDIR}${mandir}/man1/mosquitto_sub.1
-	$(INSTALL) -d ${DESTDIR}$(mandir)/man7
-	$(INSTALL) -m 644 mqtt.7 ${DESTDIR}${mandir}/man7/mqtt.7
-	$(INSTALL) -m 644 mosquitto-tls.7 ${DESTDIR}${mandir}/man7/mosquitto-tls.7
-	$(INSTALL) -d ${DESTDIR}$(mandir)/man3
-	$(INSTALL) -m 644 libmosquitto.3 ${DESTDIR}${mandir}/man3/libmosquitto.3
 
 uninstall :
 	-rm -f ${DESTDIR}${mandir}/man8/mosquitto.8
-- 
2.7.1

